### PR TITLE
WT-8473 Power8 vector intrinsic CRC32 implementation

### PR DESCRIPTION
**Pull request by kommiharibabu**
I have taken vector intrinsic approach from this library https://github.com/antonblanchard/crc32-vpmsum
Extra changes:
- fixing s_all errors
- fixing --enable-strict compilation

### CODE REVIEW

File: dist/filelist [hunk 1]
```diff
@@ -48,7 +48,7 @@ src/btree/row_key.c
 src/btree/row_modify.c
 src/btree/row_srch.c
 src/checksum/arm64/crc32-arm64.c	ARM64_HOST
-src/checksum/power8/crc32.sx		POWERPC_HOST
+src/checksum/power8/vec_crc32.c		POWERPC_HOST
```
Comment (keithbostic) [inline dist/filelist:51]: Sort order.

**Review by keithbostic — APPROVED** [review]
lgtm.

**Review by alisonfel — APPROVED** [review]
LGTM 👍

File: dist/filelist (hunk 1 above)
Comment (kommiharibabu) [reply dist/filelist]: Thanks. Fixed.

**Pull request merged**